force-app/main/default/classes/IBMDiscoveryV1Models.cls

```diff
@@ -7373,8 +7862,75 @@ public class IBMDiscoveryV1Models {
       }
 
       NluEnrichmentEmotion ret = (NluEnrichmentEmotion) super.deserialize(jsonString, jsonMap, classType);
+      NluEnrichmentEmotionBuilder retBuilder = ret.newBuilder();
 
-      return ret;
+      return retBuilder.build();
+    }
+  }
+
+  /**
+   * NluEnrichmentEmotion Builder.
+   */
+  public class NluEnrichmentEmotionBuilder {
```

Comment:
should this be part of this PR? i thought it was focused on just adding headers

Reply:
I could take that part out and do a separate PR. I included it just since I was regenerating everything but it is a pretty big change on its own.

Reply:
its not necessary, just made this giant PR even more giant :-)